model/access/systemcollection/system_collection.go

```diff
@@ -0,0 +1,90 @@
+package systemcollection
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/access"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const (
+	Version0 access.Version = 0
+	Version1 access.Version = 1
+)
+
+var ChainHeightVersions = map[flow.ChainID]access.HeightVersionMapper{
+	flow.Mainnet: access.NewStaticHeightVersionMapper(map[uint64]access.Version{
+		0:   Version0,
+		200: Version1, // TODO: set the height when we know it
```

Comment:
lets do the initial merge with:

```
0: Version1,
```

and change it to 

```
0:   Version0,
hcu_block: Version1
```
with a different commit.